### DISCUSSION TITLE
Incompatible language version

### DISCUSSION THREAD

**Discussion by rebel-ly** [tree-sitter/tree-sitter#2959] [Q&A]
![image](https://user-images.githubusercontent.com/56351033/173529223-17598e12-658b-408b-9bbb-cede1334e32a.png)

I have install the newest version tree-sitter 0.20 by commend 'pip install tree-sitter==0.20.0'
But it return the above question.
Then i try to use tree-sitter==0.19 to run it.
Same question were returned.
please help me to solve it , thanks.

**Comment by dundargoc** [comment]
I'd just use rustup/cargo tbh
